e2e/app.po.ts

```diff
@@ -103,7 +103,9 @@ export class AppPage {
 
   clickButton(buttonTitle: string): P<any> {
     log.info(`clicking button ${buttonTitle}`);
-    return this.getButton(buttonTitle).click();
+    const buttonElement = this.getButton(buttonTitle);
+    browser.wait(ExpectedConditions.presenceOf(buttonElement), 10000, 'Wait for button');
```

Comment:
I'm wondering what happens if `browser.wait` times out (in the context of https://github.com/syndesisio/syndesis-ui/issues/559).
We probably should return rejected promise with error data.
@lvydra can you check what happens if you wait for non-existent button? Is cucumber report generated or does it crash protractor and leave browser open ?